requirements.txt

```diff
@@ -1,5 +1,6 @@
 gitpython
 pyyaml
+conda ==4.1.12
 conda-build-all
 conda-build ==2.0.6
-requests !=2.12.0
+requests ==2.11.1 
```

Comment:
`2.11.1` is that last `2.11.*` release ... I can work back from `2.12.4` and see what works ...